flutter_local_notifications/lib/src/platform_specifics/ios/notification_details.dart

```diff
@@ -12,15 +12,19 @@ class IOSNotificationDetails {
   /// Specifies the name of the file to play for the notification. Requires setting [presentSound] to true. If [presentSound] is set to true but [sound] isn't specified then it will use the default notification sound.
   final String sound;
 
+  /// Show the badge number for this specific notification
```

Comment:
Please update the documentation for this so it describes how it's used. If used together with the name of the property, it's a bit awkward when read as a sentence ("the badgeNumber property shows the badge number..."). Something more like "Specifies the number to display as the app icon's badge" would be more appropriate and closer to what is described in the iOS docs